scripts/inc.writeGlobalConfig.sh

```diff
@@ -109,6 +109,16 @@ fi
 # 2. then|or read value from file
 SECONDSWIPEPAUSECONTROLS=`cat $PATHDATA/../settings/Second_Swipe_Pause_Controls`
 
+##############################################
+# Legacy mode
+# 1. create a default if file does not exist
+if [ ! -f $PATHDATA/../settings/Mode_Legacy ]; then
+    echo "ON" > $PATHDATA/../settings/Mode_Legacy
```

Comment:
If changed to "MODE_FULLID" the default has to be "OFF"

Reply:
Done, cf. d618ec0db13a052664effedcf585677624a13bb5.